PublicSDKAPITester/RevenueCatAPI.m

```diff
@@ -0,0 +1,120 @@
+    SKProduct *skp = [[SKProduct alloc] init];
+    SKProductDiscount *skpd = [[SKProductDiscount alloc] init];
+    SKPaymentDiscount *skmd = [[SKPaymentDiscount alloc] init];
+    
+    RCPackage *pack = [[RCPackage alloc] init];
+    [RCPurchases configureWithAPIKey:@"" appUserID:@""];
+    [RCPurchases configureWithAPIKey:@"" appUserID:@"" observerMode:false];
+    [RCPurchases configureWithAPIKey:@"" appUserID:@"" observerMode:false userDefaults:nil];
+    
+    [p invalidatePurchaserInfoCache];
+
+    NSDictionary<NSString *, NSString *> *attributes = nil;
+    [p setAttributes: attributes];
+    [p setEmail: @""];
+    [p setPhoneNumber: @""];
+    [p setDisplayName: @""];
+    [p setPushToken: [@"" dataUsingEncoding: NSUTF8StringEncoding]];
+    [p setAdjustID: @""];
+    [p setAppsflyerID: @""];
+    [p setFBAnonymousID: @""];
+    [p setMparticleID: @""];
+    [p setOnesignalID: @""];
+    [p setMediaSource: @""];
+    [p setCampaign: @""];
+    [p setAdGroup: @""];
+    [p setAd: @""];
+    [p setKeyword: @""];
+    [p setCreative: @""];
+    [p collectDeviceIdentifiers];
+    
+    [p purchaserInfoWithCompletionBlock:^(RCPurchaserInfo *info, NSError *error) {}];
+    [p offeringsWithCompletionBlock:^(RCOfferings *info, NSError *error) {}];
+    [p productsWithIdentifiers:@[@""] completionBlock:^(NSArray<SKProduct *> *products) { }];
+    [p purchaseProduct:skp withCompletionBlock:^(SKPaymentTransaction *t, RCPurchaserInfo *i, NSError *error, BOOL userCancelled) { }];
+    [p purchasePackage:pack withCompletionBlock:^(SKPaymentTransaction *t, RCPurchaserInfo *i, NSError *e, BOOL userCancelled) { }];
+    [p restoreTransactionsWithCompletionBlock:^(RCPurchaserInfo *i, NSError *e) {}];
+    [p syncPurchasesWithCompletionBlock:^(RCPurchaserInfo *i, NSError *e) {}];
+    [p checkTrialOrIntroductoryPriceEligibility:@[@""] completionBlock:^(NSDictionary<NSString *,RCIntroEligibility *> *d) { }];
+    [p checkTrialOrIntroductoryPriceEligibility:@[@""] completionBlock:^(NSDictionary<NSString *,RCIntroEligibility *> *r) { }];
+    [p paymentDiscountForProductDiscount:skpd product:skp completion:^(SKPaymentDiscount *d, NSError *e) { }];
+    [p purchaseProduct:skp withDiscount:skmd completionBlock:^(SKPaymentTransaction *t, RCPurchaserInfo *i, NSError *e, BOOL userCancelled) { }];
+    [p purchasePackage:pack withDiscount:skmd completionBlock:^(SKPaymentTransaction *t, RCPurchaserInfo *i, NSError *e, BOOL userCancelled) { }];
+    
+    [p createAlias:@"" completionBlock:^(RCPurchaserInfo *i, NSError *e) { }];
+    [p identify:@"" completionBlock:^(RCPurchaserInfo *i, NSError *e) { }];
+    [p resetWithCompletionBlock:^(RCPurchaserInfo *i, NSError *e) { }];
+        
+    // RCPurchasesDelegate
+    [p.delegate purchases:p didReceiveUpdatedPurchaserInfo:pi];
+    [p.delegate purchases:p shouldPurchasePromoProduct:skp defermentBlock:^(RCPurchaseCompletedBlock makeDeferredPurchase) {}];
+}
+
+
+
```

Comment:
nitpick: at most 2 line breaks

Reply:
I LOVE LINE BREAK🚀
Will fix in next pr

Reply:
✅ 
https://github.com/RevenueCat/purchases-ios/pull/592